components/OverflowMenu/react.js

```diff
@@ -35,21 +36,27 @@ export class OverflowMenu extends Component {
 
   render() {
     const { displayOverflowMenuActions } = this.state
-    const { children } = this.props
+    const { children, variant } = this.props
     return (
-      <div className={cx('overflow-menu')}>
+      <div
+        className={cx('overflow-menu', {
+          'overflow-menu--dark': ['secondary', 'dark'].includes(variant),
```

Comment:
@tarikhamilton @simonwebdesign I'm not too familiar with how this component is supposed to work so these concerns may not be relevant however I do have a couple general questions regarding this new variant prop:

1. Do you think we would ever want the overflow menu content to also use the variant styles and if so what does that look it? i.e. dark variant button and dark style for the overflow menu.
2. If yes to the above, would we ever need any of the following variants?
    - dark button with light popup
    - light button with dark popup
    If so, we may want to consider updating the prop names to be more specific since dark seems like both the button & overflow menu.

Reply:
We do want the menu to be dark too, but it isn't designed yet. I was waiting for Simon to do it, but we can always update the library incrementally.